src/main/java/meteordevelopment/meteorclient/systems/modules/render/NoRender.java

```diff
@@ -113,6 +113,13 @@
             .build()
     );
 
+    private final Setting<Boolean> onRenderSuggestion = sgOverlay.add(new BoolSetting.Builder()
```

Comment:
`onRenderSuggestion` makes no sense for a name.
`noCommandSuggestions` would be better.

Reply:
Okay now it should be done